client/src/App.tsx

```diff
@@ -169,6 +174,14 @@ function App() {
 		setChatMessages((chatMessages) => chatMessages.concat(newMessage));
 	}, []);
 
+	const changeBackground = useCallback((newBackgroundName: string) => {
```

Comment:
this is extra logic, why not just `setBackgroundName(newBackgroundName)?`
You only need to use this format of setSomeStateVariable(variable => { ... }) if you actually need to use the current state. We use this format for setEmojis( emojis => { ... } ) because emojis is an array and we add to the array, in this case we don't care about backgrounds current state, all we care about is changing it to the new state